### DISCUSSION TITLE
UI reshuffle

### DISCUSSION THREAD

**Discussion by johnaohara** [Hyperfoil/Horreum#742] [Ideas]
One of the UX challenges currently is each page in the UI is more-or-less standalone.  This means that user find themselves having to reselect folders->test when they click on a link.

I have been playing with changes to the UI, so that once a test is selected, everything relevant to that test is accessible without having to re-select the test again

I have also hidden the concept of a "Run" from the UI and just display datasets, although the orginal run data is still accessible if you click on a particular dataset

![image](https://github.com/Hyperfoil/Horreum/assets/959822/046497ac-71c7-4b7d-a8b7-c7993229b60f)

I have a branch is anyone wants to play around with it: https://github.com/johnaohara/Horreum/tree/horreum-ui-rebase

**Comment by stalep** [comment]
This would be a very welcomed improvement to the usability of the UI. A huge +1!
